test/e2e-test/rollout_plan_test.go

```diff
@@ -585,7 +606,73 @@ var _ = Describe("Cloneset based rollout tests", func() {
 					return ""
 				}
 				return clonesetOwner.Kind
-			}, time.Second*30, time.Second).Should(BeEquivalentTo(v1alpha2.ApplicationContextKind))
+			}, time.Second*30, time.Second).Should(BeEquivalentTo(v1beta1.ResourceTrackerKind))
+	})
+
+	It("Test rollout will update same name trait", func() {
+		CreateClonesetDef()
+		CreateIngressDef()
+		applySourceApp("app-with-ingress-source.yaml")
+		By("Apply the application rollout go directly to the target")
+		Expect(common.ReadYamlToObject("testdata/rollout/cloneset/appRollout.yaml", &appRollout)).Should(BeNil())
```

Comment:
```suggestion
                 appRollout = v1beta1.AppRollout{}
		Expect(common.ReadYamlToObject("testdata/rollout/cloneset/appRollout.yaml", &appRollout)).Should(BeNil())
```
appRollout is shared by several cases, remember to reset it in each case.